docs/docs/guides/integrating-with-rill.md

```diff
@@ -0,0 +1,173 @@
+3. Administrative access to your Rill organization
+
+:::info
+This guide can be completed with User tokens. These tokens are tied to your personal user and access permissions. They are useful for local scripting and experimentation. We don't recommend for production use.
+:::
+
+## Example: Managing Users with a Rill Service Account
+
+This example demonstrates how to create a service account, use it to manage users with custom attributes, and then clean up all resources. This is useful for automating user provisioning and management in your Rill organization.
+
+### Step 1: Create a Service Account
+
+First, create a service account with organization admin permissions and custom attributes:
+
+```bash
+# Create a service account with custom attributes
+rill service create user-management-service \
+  --org-role admin \
+  --attributes '{"department": "engineering", "environment": "production"}'
+
+Service created: user-management-service
+Token: rill_svc_A1B2C3D4E5F6G7H8I9J0K1L2M3N4O5P6Q7R8S9T0U1V2W3X4Y5Z6
+```
+
+### Step 2: Issue a Service Token
+
+Generate an authentication token for the service account:
+
+```bash
+# Issue a token for the service account
+rill service token issue user-management-service
+Token: rill_svc_A1B2C3D4E5F6G7H8I9J0K1L2M3N4O5P6Q7R8S9T0U1V2W3X4Y5Z6
+```
+
+### Step 3: Use Service Account to Add a User with Custom Attributes
+
+Now use the service account token to add a user to your organization:
+
+```bash
+# Add a user to the organization using the service account
+curl -X POST "https://admin.rilldata.com/v1/orgs/example/members" \
+     -H "Authorization: Bearer rill_svc_A1B2C3D4E5F6G7H8I9J0K1L2M3N4O5P6Q7R8S9T0U1V2W3X4Y5Z6" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "email": "newuser@example.com",
+           "role": "viewer",
+         }'
+```
+
+**Expected Response:**
+```json
+{
+  "pendingSignup": true
+}
+```
+
+The user will receive an email invitation to join the Rill organization as a viewer.
+
+### Step 4: Create Ephemeral User Tokens with Custom Attributes
+
+Use the service account to create short-lived tokens for specific users with custom attributes:
+
+```bash
+# Create an ephemeral user token with custom attributes
+curl -X POST "https://admin.rilldata.com/v1/orgs/example/projects/my-project/credentials" \
+     -H "Authorization: Bearer rill_svc_A1B2C3D4E5F6G7H8I9J0K1L2M3N4O5P6Q7R8S9T0U1V2W3X4Y5Z6" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "userEmail": "newuser@example.com",
+           "ttlSeconds": 3600,
+           "attributes": {
+             "department": "marketing",
+             "role": "analyst",
+             "customField": "value123"
+           }
+         }'
+```
+
+**Expected Response:**
+```json
+{
+  "accessToken": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9...",
+  "instanceId": "inst_1234567890abcdef",
+  "host": "runtime.rilldata.com",
+  "grpcPort": 443
+}
+```
```

Comment:
I would not cover runtime token issuance in this example, it confounds the `admin` APIs and `runtime` APIs, which requires a deeper explainer about the difference between the two layers, which is worthy of a dedicated docs page if anything (some of this is already somewhat covered by the runtime API access docs page).